packages/bpk-component-loading-button/src/BpkLoadingButton-test.js

```diff
@@ -25,75 +25,90 @@ import BaggageIcon from '../../bpk-component-icon/sm/baggage';
 
 import BpkLoadingButton, { ICON_POSITION } from './BpkLoadingButton';
 
+const error = jest.spyOn(console, "error").mockImplementation(() => {});
+
 describe('BpkLoadingButton', () => {
+  beforeEach(() => {
+    error.mockReset();
+  })
   it('should render correctly', () => {
     const { asFragment } = render(
       <BpkLoadingButton>My button</BpkLoadingButton>,
     );
     expect(asFragment()).toMatchSnapshot();
+    expect(error).not.toHaveBeenCalled();
   });
 
   it('should render correctly with a "href" attribute', () => {
     const { asFragment } = render(
       <BpkLoadingButton href="#">My button</BpkLoadingButton>,
     );
     expect(asFragment()).toMatchSnapshot();
+    expect(error).not.toHaveBeenCalled();
   });
 
   it('should render correctly with a "primaryOnDark" attribute', () => {
     const { asFragment } = render(
       <BpkLoadingButton primaryOnDark>My button</BpkLoadingButton>,
```

Comment:
Any reason our tests are still doing the old way and not actually using it as it should be?

With one test just working with an attribute?

Reply:
The api of `BpkLoadingButton` hasn't changed, just the underlying implementation.